guides/v2.2/cloud/env/variables-deploy.md

```diff
@@ -261,6 +261,9 @@ stage:
 
 ### `SCD_EXCLUDE_THEMES`
 
+{: .bs-callout .bs-callout-warning }
+The `SCD_EXCLUDE_THEMES` environment variable is deprecating soon. Use the [SCD_MATRIX variable](#scd_matrix) to control theme configuration.
```

Comment:
@oshmyheliuk   Should the notes in the build and deploy variable documentation say that SCD_EXCLUDE_THEMES "is deprecated" instead of "deprecating soon"?

Reply:
Yes, it's deprecated from 2002.0.16